src/posit/connect/users.py

```diff
@@ -151,3 +151,10 @@ def update(self) -> User:
 
     def delete(self) -> None:
         raise NotImplementedError()
+
+    def count(self) -> int:
+        response = self.session.get(self.url)
+        result: dict = response.json()
+        if "total" not in result:
+            raise RuntimeError("'total' not found in response body")
+        return result.get("total", 0)
```

Comment:
Can't see that we'd break this API, and even if we did, I think it's fine to let the `KeyError` bubble up.

```suggestion
        return response.json()["total"]
```